03-pipefilter.md

```diff
@@ -16,46 +16,53 @@ Now that we know a few basic commands,
 we can finally look at the shell's most powerful feature:
 the ease with which it lets us combine existing programs in new ways.
 We'll start with a directory called `molecules`
-that contains six files describing some simple organic molecules.
-The `.pdb` extension indicates that these files are in Protein Data Bank format,
-a simple text format that specifies the type and position of each atom in the molecule.
+so lets go into that directory and take a look at that directory:
 
 ~~~ {.bash}
-$ ls molecules
+$ cd molecules
+~~~
+~~~ {.bash}
+$ ls
 ~~~
 ~~~ {.output}
-cubane.pdb    ethane.pdb    methane.pdb
-octane.pdb    pentane.pdb   propane.pdb
+aldrin.pdb
+ammonia.pdb
+ascorbic-acid.pdb
+...
+vanillin.pdb
+vinyl-chloride.pdb
+vitamin-a.pdb
 ~~~
 
-Let's go into that directory with `cd` and run the command `wc *.pdb`.
-`wc` is the "word count" command:
-it counts the number of lines, words, and characters in files.
-The `*` in `*.pdb` matches zero or more characters,
-so the shell turns `*.pdb` into a complete list of `.pdb` files:
+The `.pdb` extension indicates that these files are in Protein Data Bank format,
+a simple text format that specifies **at each line** the type and position of **one** atom in the molecule.
+We can get only the files with the `.pdb` extension.
 
 ~~~ {.bash}
-$ cd molecules
-$ wc *.pdb
+$ ls *.pdb
 ~~~
 ~~~ {.output}
-  20  156 1158 cubane.pdb
-  12   84  622 ethane.pdb
-   9   57  422 methane.pdb
-  30  246 1828 octane.pdb
-  21  165 1226 pentane.pdb
-  15  111  825 propane.pdb
- 107  819 6081 total
+aldrin.pdb
+ammonia.pdb
+ascorbic-acid.pdb
+...
+vanillin.pdb
+vinyl-chloride.pdb
+vitamin-a.pdb
 ~~~
 
+The `*` in `*n.pdb` matches zero or more characters,
+so the shell turns `*nol.pdb` into a complete list
```

Comment:
These three lines mention *n.pdb, *nol.pdb and *.pdb. I'm confused as to whether that is the intention?